monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,66 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbPodRecovering
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) == 3
+    for: 1h
+    labels:
+      severity: warning
+    annotations:
+      description: "MongoDB pod `{{ $labels.pod }}` has been stuck in the 'RECOVERING' state for the past hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
+      summary: MongoDB node recovering for too long
+
+  - alert: MongoDbPodRecovering
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1d]) == 3
+    for: 1d
```

this will require 2 d to trigger, and require lots of resources to evaluate.

this is especially critical here, since a query over a 1d period will be very slow...

```suggestion
      mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*"} == 3
    for: 1d
```